servicex/servicex_client.py

```diff
@@ -45,12 +45,19 @@
 from make_it_sync import make_sync
 from servicex.databinder_models import ServiceXSpec, General, Sample
 from collections.abc import Sequence
+from enum import Enum
 import traceback
 
 T = TypeVar("T")
 logger = logging.getLogger(__name__)
 
 
+class ProgressBarFormat(str, Enum):
+    default = "default"
+    compact = "compact"
```

Comment:
Add `expanded`, and have `default` be the same as `expanded` - that makes this stable against future changes for people that want it.

Reply:
OK